### ISSUE TITLE
Setting timezone for embbeded postgresql

### ISSUE THREAD

**Issue by swapy** [zonkyio/embedded-database-spring-test#233]
Hello,

Can we set default timezone for this embedded database? If yes, can you let me know how?

**Comment by tomix26** [comment]
Hi @swapy, thanks for the question. You can set it using the following configuration property:
```
zonky.test.database.postgres.server.properties.timezone=GMT
```
The server properties correspond to the options in the [postgresql.conf](https://github.com/postgres/postgres/blob/master/src/backend/utils/misc/postgresql.conf.sample) file. So you can change other settings as well.

**Comment by swapy** [comment]
Thank you very much @tomix26 

I tried setting GMT as well as american time but it did not work

![image](https://user-images.githubusercontent.com/1929023/229757297-bc347138-1ff7-4f52-bb54-60dd62bd08aa.png)

**Comment by tomix26** [comment]
Yes, you're right, it doesn't work. Maybe it's because the server parameters are now passed as command-line parameters instead of editing the `postgresql.conf` file. But according to the documentation [here](https://www.postgresql.org/docs/12/datatype-datetime.html), even the command-line parameters should work.
> The [TimeZone](https://www.postgresql.org/docs/12/runtime-config-client.html#GUC-TIMEZONE) configuration parameter can be set in the file postgresql.conf, or in any of the other standard ways described in [Chapter 19](https://www.postgresql.org/docs/12/runtime-config.html).

I'll have to investigate the problem more and I assume it will require some changes in the library. Note that even the `ALTER SYSTEM SET timezone = 'America/Los_Angeles'` sql command doesn't work for me, which is really weird.

**Comment by tomix26** [comment]
After upgrading to `io.zonky.test:embedded-postgres:2.2.0` (https://github.com/zonkyio/embedded-database-spring-test/issues/307), a new configuration option `EmbeddedPostgres.Builder#setDataDirectoryCustomizer` (https://github.com/zonkyio/embedded-postgres/issues/144) will be available. This option will allow you to make any changes to PostgreSQL configuration files. This should be enough to resolve or work around the issue reported above.
